app/email.go

```diff
@@ -201,17 +204,18 @@ func (es *EmailService) sendWelcomeEmail(userId string, email string, verified b
 		map[string]interface{}{"SiteName": es.srv.Config().TeamSettings.SiteName,
 			"ServerURL": serverURL})
 
-	bodyPage := es.newEmailTemplate("welcome_body", locale)
+	bodyPage := es.newEmailTemplate("welcome", locale)
```

Comment:
I know is no longer "body" because you are generating the whole HTML, but as I said in other comment, we have users that relays on modifications on this files, I think i prefer to keep the filename exactly the same for backward compatibility.

Reply:
Yes done.